src/index.ts

```diff
@@ -22,3 +23,7 @@ export * from './services/contracts/EosioTokenContract';
 export * from './services/errors';
 export * from './userApps';
 export * from './communication';
+export * from './util/message';
+
+export { createSigner } from '@tonomy/antelope-ssi-toolkit';
```

Comment:
this should not be exported from an imported package. if someone wants this they can do it directly